app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MechanicsController < ApplicationController
+  def index
+  end 
+
+  def show
+    @mechanic = Mechanic.find(params[:id])
+  end
+
+  def update
```

Comment:
Good intuition - this can indeed be in a joins controller! Consider routing that form in the show page to a controller called `RideMechanicsController` and specifically to a `create` action. This is a more RESTful path because what you're doing is not updating attributes of a mechanic, but rather *creating* a joins record in the ride_mechanics table.